### PR TITLE
Push popup to top left and update content

### PR DESCRIPTION
**Pull request by prokarius**
This is meant to make our mocks in line with the presentation scripts.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -15,15 +15,20 @@ function mouseoverEvent (){
...
-      imageUrl: "https://i.imgur.com/EHuPqnT.png",
-      imageWidth: 300,
-      imageHeight: 100,
+      imageUrl: "https://i.ibb.co/4my0R7M/Confirm.png",
       imageAlt: 'Budget image'
+    }).then((result) => {
+      Sweetalert2.fire({
+        animation: true,
+        showConfirmButton: false,
+        timer: 5000,
```
Comment (Waynerbee) [inline script.js]: We should have a longer time. Maybe set to 8 seconds.

**Review by Waynerbee — CHANGES_REQUESTED** [review]
LGTM, but address timing issues.

File: script.js (hunk 1 above)
Comment (prokarius) [reply script.js]: :+1:

**Review by Waynerbee — APPROVED** [review]
LGTM

**Pull request merged**